tutorials/hyperformula-integration.html

```diff
@@ -0,0 +1,257 @@
+          ["Sum", "Average", "Average", "Sum", "Sum"],
+          ["=SUM(A2:A100)", "=AVERAGE(B2:B100)", "=AVERAGE(C2:C100)", "=SUM(D2:D100)", "=SUM(E2:E100)"]
+        ];
+
+        var container = document.getElementById('example-data-grid');
+        new Handsontable(container, {
+          data: data,
+          formulas: {
+            engine: HyperFormula,
+          },
+          colHeaders: ["Qty", "Unit price", "Discount", "Freight", "Total due (fx)"],
+          fixedRowsBottom: 2,
+          height: 500
+        });</script>
+    </div>
+
+    <h3 id="initialization-methods">Initialization methods</h3>
+    <p>There're multiple ways of initializing the plugin, you can select the most convenient one depending on your use case.</p>
+
+    <p>In all cases it is required to either pass in the <code>HyperFormula</code> object or a HyperFormula instance:</p>
+    <pre><code class="hljs lang-javascript">import { HyperFormula } from 'hyperformula';</code></pre>
+    <p>There are also other installation methods available, check out <a href="https://handsontable.github.io/hyperformula/guide/client-side-installation.html">HyperFormula's installation documentation</a>.</p>
+
+
+    <!-- todo should we write demos for these? -->
+    <h4 id="passing-hf-to-hot">Passing the HyperFormula class/instance to Handsontable</h4>
+    <pre><code class="hljs lang-javascript">{
+  formulas: {
+    engine: HyperFormula,
+    // [plugin configuration]
+  }
+}</code></pre>
+    <p>or</p>
+    <pre><code class="hljs lang-javascript">{
+  formulas: {
+    engine: {
+      hyperformula: HyperFormula, // or `engine: hyperformulaInstance`
+    },
+    // [plugin configuration]
+  }
+}</code></pre>
+
+    <h4 id="single-hot-external-hf">Single Handsontable instance with an external HyperFormula instance</h4>
+    <pre><code class="hljs lang-javascript">const hyperformulaInstance = HyperFormula.buildEmpty({})
+
+{
+  formulas: {
+    engine: hyperformulaInstance
+  }
+}</code></pre>
+
+    <h4 id="multiple-independent-hot">Multiple independent Handsontable instances</h4>
+    <pre><code class="hljs lang-javascript">// Instance 1
+{
+  formulas: {
+    engine: HyperFormula,
+    // [plugin configuration]
+  }
+}
+
+// Instance 2
+{
+  formulas: {
+    engine: HyperFormula,
+    // [plugin configuration]
+  }
+}</code></pre>
+
+    <h4 id="multiple-hf-with-shared-hf">Multiple Handsontable instances with a shared HyperFormula instance</h4>
+    <pre><code class="hljs lang-javascript">// Instance 1
+{
+  formulas: {
+    engine: HyperFormula,
+    sheetName: 'Sheet 1'
+    // [plugin configuration]
+  }
+}
+
+// Instance 2
+{
+  formulas: {
+    engine: hot1.getPlugin('formulas').engine,
+    sheetName: 'Sheet 2'
```

Comment:
Probably it's safer not to use special characters. In this case an empty space. 
```suggestion
    sheetName: 'Sheet2'
```